index.js

```diff
@@ -1,57 +1,46 @@
-function gampee(ecData) {
+var mappings = require("./lib/mappings"),
+	validate = require("./lib/validate"),
+	utils = require("./lib/utils");
 
-	if (!ecData) {
-		throw new TypeError();
+function gampee(actionList, strict) {
+
+	if (!actionList) {
+		throw new TypeError("actionList is required");
 	}
 
-	if (!Array.isArray(ecData)) {
-		ecData = [ecData];
+	if (!Array.isArray(actionList)) {
+		actionList = [actionList];
 	}
 
+	var validationCallback = strict ? utils.throwOnError : utils.consoleWarnOnError;
```

Comment:
I did exactly that in my first pass (never checked in, though). This will work better in the [quite near] future.